packages/legend-studio/src/models/protocols/pure/v1/transformation/pureGraph/to/helpers/V1_RelationalConnectionBuilderHelper.ts

```diff
@@ -123,6 +127,16 @@ export const V1_processDatasourceSpecification = (
     snowflakeSpec.quotedIdentifiersIgnoreCase =
       protocol.quotedIdentifiersIgnoreCase;
     return snowflakeSpec;
+  } else if (protocol instanceof V1_BigQueryDatasourceSpecification) {
+        assertNonEmptyString(
```

Comment:
Where is the assertion for `defaultDataset`?

Reply:
It is an optional property. If specified it is used as the default dataset.